### DISCUSSION TITLE
Where to solder 4pin cables?

### DISCUSSION THREAD

**Discussion by olskar** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#333] [Q&A] [answered]
Hello! 
I have a 4pin pump and PCB V1.

I understand the cables should be soldered to he and he but which one to which?

Also, I'm not sure I understand the colours referenced in instructions, the cables in my pump does not have any colours

**Comment by visualapproach** [comment]
Hi, the most important part of the instructions is that you need to figure out which wire is 5V and GND. The other two can be trial and error.

**Reply by olskar** [reply]
Thank you @visualapproach ! Does it matter if I solder jst female connector or male connector to H3 or H4 ?

**Reply by visualapproach** [reply]
Yes, it will be difficult to join two of the same gender. One must go to the cio (pump). Look at the PCB where it says CIO. Then look at the connector coming from the pump. Those should mate.

**Comment by chunkysteveo** [comment]
The cables may be white with red dashes and dots... look in some closed issues for "dash" and I am sure I had a reference image for what my 4 wire was...

**Comment by chunkysteveo** [comment, marked as answer]
![20220111_141310](https://user-images.githubusercontent.com/14663002/179057234-14454c17-f514-481b-b002-68b5a1f3c845.jpg)
 
This image should help. Ignore the coloured cables as my red is NOT 5V, etc!

**Comment by olskar** [comment]
Thank you @visualapproach and @chunkysteveo for the image!

Lay z spa vegas 4-wire version is working if you want to add that to the instructions :)

**Reply by visualapproach** [reply]
Cool! Do you know what model number it is? Like 54123 or similar usually found on a sticker on the pump

**Reply by visualapproach** [reply]
And also what model you defined

**Reply by olskar** [reply]
Model is 54112, works with NO54123 defined

**Comment by derpaddy89** [comment]
unfortunately doesn't work for me. use the current v4 code. Does the display have to be switched on manually? is like a standby.
